### PR TITLE
[Mobile - Theme] Fix neutralForeground1 in Tester

### PR DESCRIPTION
**Pull request by rohanprasadofficial**
### Platforms Impacted
- [x] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [x] android

### Description of changes

Fixes neutralForeground1 in Tester App.  
'red' color was applied to neutralForeground1 across Fluent Tester. 

Regression of this pr : #2668 

### Verification

(how the change was tested, including both manual and automated tests)

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| MicrosoftTeams-image (35) | MicrosoftTeams-image (36) |

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Review by lyzhan7 — APPROVED** [review]

**Comment by lyzhan7** [comment]
Confirmed this issue repros on iOS and this fix works for iOS as well

**Comment by rohanprasadofficial** [comment]
will wait for approval from @ankraj12

**Review by ankraj12 — APPROVED** [review]
Thanks for fixing this

**Pull request merged**
